packages/web-react/README.md

```diff
@@ -97,11 +99,26 @@ All Spirit components accept a set of props that can be used to control their ou
 These props accept a spacing token (eg. `space-100`), `auto` or an object with breakpoint keys and spacing token
 values or `auto`. We use these props to set global CSS utility classes on the root element of the component.
 
+#### Hide Props
+
+- `hideOn` - Hide the component only at specific breakpoint(s)
+- `hideFrom` - Hide the component from a specific breakpoint and up
```

Comment:
?

```suggestion
- `hideOn` - Hides the component only at specific breakpoint(s)
- `hideFrom` - Hides the component from a specific breakpoint and up
```

Reply:
imho not, it is "to hide the component do this"